docs/qa/MainQA.md

```diff
@@ -0,0 +1,70 @@
+| Main42 | Check if languages that are written from right to left are used that placeholder appears in top right corner                                                         |        |                     |             |                              |
+| Main43 | Check that for all supported languages placeholder is written correctly and is written in chosen interface language                                                  |        |                     |             |                              |
+| Main44 | Ensure that placeholder is in gray color                                                                                                                             |        |                     |             |                              |
+| Main45 | Confirm that placeholder disappears when first letter is written                                                                                                     |        |                     |             |                              |
+| Main46 | Verify that placeholder appears again if input field is empty (if there are spaces, it is not considered empty)                                                      |        |                     |             |                              |
+| Main47 | Try input entering special characters, symbols, and non-Latin characters to ensure the input field handles them correctly.(проверка, 1234, #!? 😋 我吃晚饭了)             |        |                     |             |                              |
+| Main48 | Verify that any validation messages or alerts appear when users enter incorrect or invalid input                                                                     |        |                     |             |                              |
+| Main49 | Check that "x" button appears in right up corner of text field when any letter is written in the text field                                                          |        |                     |             |                              |
+| Main50 | Check that "x" button disappears when textfield is made empty manually by user                                                                                       |        |                     |             |                              |
+| Main51 | Check that when "x" button is pressed that all text is removed from text field and x button disappears                                                               |        |                     |             |                              |
+| Main53 | Test the input field's character limit, if any, to ensure users cannot exceed the allowed length.                                                                    |        |                     |             | Currently limit is 500 words |
```

Comment:
_Currently limit is 500 words_  - it looks more like requirement, not commentary.